source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+
+<b>Technical Description</b>
+
+N/A
+
+##G0017
+
+<b>Functional Description</b>
+
+'State of seals' = 1 in case that Seals are in good state (i.e. present and not damaged)<br/>'State of seals' = 0 in case that Seals are not in good state (i.e. not present or damaged).
+
+<b>Technical Description</b>
+
+N/A
+
+##G0018
+
+<b>Functional Description</b>
+
+IF &lt;COMPETENT CUSTOMS OFFICE AT DEPARTURE&gt; is EQUAL to &lt;CUSTOMS OFFICE OF DEPARTURE&gt;<br/>THEN &lt;CD144C-CUSTOMS OFFICE OF DEPARTURE&gt; = "O" <br/>ELSE &lt;CD144C-CUSTOMS OFFICE OF DEPARTURE&gt; = "R"
+
+<b>Technical Description</b>
+
+N/A
+
+##G0020
+
+<b>Functional Description</b>
+
+Refers to the mode of transport corresponding to the active means of transport which is expected to be used on exit from or entry into the Safety and Security area.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0021
+
+<b>Functional Description</b>
+
+The value '0' (zero) is a valid number in this Data Item, as per applicable XSD pattern.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0022
+
+<b>Functional Description</b>
+
+This D.I. corresponds to the D.I. Security of the Export Declaration.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0023
+
+<b>Functional Description</b>
+
+The D.I. will be filled in with new value amending initial declaration in case of incident
+
+<b>Technical Description</b>
+
+N/A
+
+##G0024
+
+<b>Functional Description</b>
+
+D.I. will be filled if the value is provided.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0025
+
+<b>Functional Description</b>
+
+The value of D.I. &lt;CONSIGNMENT-HOUSE CONSIGNMENT.Security indicator from export declaration&gt; will be equal to D.I. &lt;CC191C-AES RESULTS-EXPORT OPERATION.Security&gt;.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0026
+
+<b>Functional Description</b>
+
+This D.G./D.I. can be used only  <br/>         - in case of Export followed by transit (maximum one MRN included per one House Consignment) <br/>     AND <br/>         - if the &lt;TRANSIT OPERATION.Additional Declaration Type&gt; is EQUAL to 'A'.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0029
+
+<b>Functional Description</b>
+
+The D.I. will be filled in with new value amending initial declaration in case of incident. In case goods were initially not containerized and are placed in a container, or the initial container is replaced by another container then &lt;CONSIGNMENT-INCIDENT-TRANSHIPMENT.Container indicator&gt; is equal to '1' else &lt;CONSIGNMENT-INCIDENT-TRANSHIPMENT.Container indicator&gt; is equal to '0'.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0030
+
+<b>Functional Description</b>
+
+If a consignment is moving from one Member State to another Member State via a third country which is not in set of CL009 (CountryCodesCommonTransit) then a &lt;CUSTOMS OFFICE OF TRANSIT (DECLARED)&gt; shall be declared and located in the specific Member States. 
+
```

Comment:
Multiple problems here (via, which, then).

Suggestion: If a consignment is moving from one Member State to another Member state by means of a third country that is not set in CL009 (CountryCodesCommonTransit), a &lt;CUSTOMS OFFICE OF TRANSIT (DECLARED)&gt; shall be declared and located in the specific Member States.